src/components/app/components/Footer.js

```diff
@@ -1,19 +1,15 @@
 import React from 'react';
 import { Layout } from 'antd';
 
-import { FOOTER_DATA, FOOTER_CLASSNAMES } from './constants';
+import { FOOTER_DATA, FOOTER_CLASSNAMES } from '../constants';
 
 const { Footer } = Layout;
 const { CONTAINER, DYNAMIC_FOOTER, ROW } = FOOTER_CLASSNAMES;
 
 const footers = FOOTER_DATA.map(footer => (
     <div className={DYNAMIC_FOOTER} key={footer.key}>
         <h3>{footer.header}</h3>
-        {
-            footer.items.map(item => (
-                <div key={item.title}>{item.title}</div>
-            ))
-        }
+        {footer.items.map(item => <div key={item.title}>{item.title}</div>)}
```

Comment:
Why did you make this one line of code

Reply:
if it can fit on one line, make it fit on one line, especially if it doesn't make the code harder to read. But that a balance that you have to strike for yourself. I personally prefer one lines because I can read it like english.

Reply:
OK

Reply:
me too
i prefer one line code